plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -532,6 +758,17 @@ fu_cros_ec_usb_device_write_firmware (FuDevice *device,
 		return FALSE;
 	}
 
+	if (self->in_bootloader) {
+		fu_device_set_custom_flags (device, "rw-written");
+	} else if (fu_device_has_custom_flag (device, "rw-written")) {
+		fu_device_set_custom_flags (device, "ro-written,rw-written");
+	} else {
+		fu_device_set_custom_flags (device, "ro-written");
+	}
```

Comment:
just a style nit in these - no need for curly braces if you've only got one line in each if block.  It will be more readable.